.gitignore

```diff
@@ -1,9 +1,44 @@
+# User-specific files
```

Comment:
This seems overreaching, I think you must have pulled it from some presets repo. Let’s keep the ignores to what the projects actually produce?